crates/pop-parachains/src/call/metadata/mod.rs

```diff
@@ -82,7 +82,6 @@ pub fn parse_chain_metadata(client: &OnlineClient<SubstrateConfig>) -> Result<Ve
 											// dispatchable function as unsupported rather than
 											// error.
 											is_supported = false;
-											parsed_params.clear();
```

Comment:
When parsing the chain metadata, we can't process arguments that are nested calls (e.g., `sudo` or `proxy`), they are not supported via `pop call`. I removed this line to ensure we can still parse the remaining arguments, allowing us to manually construct the `proxy` call (see `construct_proxy_extrinsic`).